analog-clock.html

```diff
@@ -73,9 +73,10 @@
     tick: function() {
       var self = this;
       var d = new Date();
+      var minutes = d.getMinutes() / 60
```

Comment:
this is not quite minutes right :) it is a fraction of 60 :) not sure the best name for it ha